### ISSUE TITLE
Design Intent - Accessibility Feedback - Public Websites: PACT Act Wizard

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#61025]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- It sounds like you're already way ahead of me on my feedback! But I'll post it anyway! 🙂 

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should:** Use the [sub task pattern](https://design.va.gov/patterns/help-users-to/complete-a-sub-task) for moving through this workflow. Besides the benefits for assistive technology compatibility, breaking things up one-thing-per-page should help with cognitive load for each individual page.
- **Consider:** For this workflow, cognitive load is definitely going to be the toughest challenge. Besides being deliberate in how you ask questions, consider warning Veterans about what kind of and how much information you'll be asking for (eg. "We'll be asking you between X and Y questions about when and where you served" etc). It may even be appropriate to provide updates along the way based on responses, like "Since you said [response], we're going to ask you some extra questions." Basically anything that can help a Veteran prepare for the length and scope of the questions they'll need to answer---especially if a future iteration eventually asks about conditions in addition to dates and locations.

**Comment by cindymerrill** [comment]
FYI @DanielleThierryUSDSVA: The above feedback includes content advice that we could consider addressing now before usability research, if you want:

_**Consider**: For this workflow, cognitive load is definitely going to be the toughest challenge. Besides being deliberate in how you ask questions, consider warning Veterans about what kind of and how much information you'll be asking for (eg. "We'll be asking you between X and Y questions about when and where you served" etc). It may even be appropriate to provide updates along the way based on responses, like "Since you said [response], we're going to ask you some extra questions." Basically anything that can help a Veteran prepare for the length and scope of the questions they'll need to answer---especially if a future iteration eventually asks about conditions in addition to dates and locations.: For this workflow, cognitive load is definitely going to be the toughest challenge. Besides being deliberate in how you ask questions, consider warning Veterans about what kind of and how much information you'll be asking for (eg. "We'll be asking you between X and Y questions about when and where you served" etc). It may even be appropriate to provide updates along the way based on responses, like "Since you said [response], we're going to ask you some extra questions." Basically anything that can help a Veteran prepare for the length and scope of the questions they'll need to answer---especially if a future iteration eventually asks about conditions in addition to dates and locations._

**Comment by cindymerrill** [comment]
@DanielleThierryUSDSVA Just noticed that this was feedback from earlier, not from today's Midpoint Review.

**Comment by FranECross** [comment]
Slack message to Amanda, Danielle and Laura regarding this tickets disposition: https://dsva.slack.com/archives/C52CL1PKQ/p1706890601440849

**Comment by laflannery** [comment]
Should feedback was incorporated we are using the subtask pattern

Consider feedback is more content focused which would be something that Danielle is responsible for and as this is very much dependent on other stakeholders the reasoning why this was no incorporated would have come from them

**Comment by FranECross** [comment]
As noted above, the Should feedback was incorporated; the Consider feedback won't be addressed right away, but Danielle is reviewing all content with stakeholders and another ticket will be created to track future updates.
